x-pack/plugins/enterprise_search/public/applications/enterprise_search_content/components/new_index/select_connector/select_connector.tsx

```diff
@@ -104,83 +103,22 @@ export const SelectConnector: React.FC = () => {
-          );
-        }}
-      >
-        <EuiFormFieldset
-          legend={{
-            children: (
-              <EuiCallOut
-                size="m"
-                title={i18n.translate(
-                  'xpack.enterpriseSearch.content.indices.selectConnector.callout.title',
-                  { defaultMessage: 'Elastic connectors' }
-                )}
-                iconType="iInCircle"
-              >
-                <p>
-                  <FormattedMessage
-                    id="xpack.enterpriseSearch.content.indices.selectConnector.description.textcloud"
-                    defaultMessage="{native} are available directly within Elastic Cloud deployments. No additional infrastructure is required. {learnMore}"
-                    values={{
-                      learnMore: (
-                        <EuiLink target="_blank" href={docLinks.connectorsNative}>
-                          {LEARN_MORE_LINK}
-                        </EuiLink>
-                      ),
-                      native: (
-                        <b>
-                          {i18n.translate(
-                            'xpack.enterpriseSearch.content.indices.selectConnector.callout.description.native',
-                            { defaultMessage: 'Native connectors' }
-                          )}
-                        </b>
-                      ),
-                    }}
-                  />
-                  <br />
-                  <br />
-                  <FormattedMessage
-                    id="xpack.enterpriseSearch.content.indices.selectConnector.description.selfManaged.text"
-                    defaultMessage="Deploy connectors on your own infrastructure as {connectorsClient}. You can also customize existing connector clients, or build your own using our connector framework. {learnMore}"
-                    values={{
-                      connectorsClient: (
-                        <b>
-                          {i18n.translate(
-                            'xpack.enterpriseSearch.content.indices.selectConnector.callout.description.connectorsClient',
-                            { defaultMessage: 'connector clients' }
-                          )}
-                        </b>
-                      ),
-                      learnMore: (
-                        <EuiLink target="_blank" href={docLinks.connectorsClients}>
-                          {LEARN_MORE_LINK}
-                        </EuiLink>
-                      ),
-                    }}
-                  />
-                </p>
-              </EuiCallOut>
-            ),
-          }}
-        >
-          <EuiFlexGroup>
-            {/* Only facet is for native connectors, so only show facets if we can show native connectors */}
-            {hasNativeAccess && (
+      <EuiFlexGroup>
+        {/* Only facet is for native connectors, so only show facets if we can show native connectors */}
+        {hasNativeAccess && (
```

Comment:
hasNativeAccess check will disable everything rather than removing the items on certain filtering options. 

This will be done as a followup PR.